backend/amd64/simd_proc.ml

```diff
@@ -0,0 +1,121 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                      Max Slater, Jane Street                           *)
+(*                                                                        *)
+(*   Copyright 2024 Jane Street Group LLC                                 *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+[@@@ocaml.warning "+a-4-30-40-41-42"]
```

Comment:
```suggestion
[@@@ocaml.warning "+a-30-40-41-42"]
```
Given the nature of the bug, I would rather
not disable this warning.

Reply:
(also I think this is an old warning stanza from an old version of flambda2, we can probably remove it now since the build system now sets reasonable warnings)

Reply:
I've added back all the unnecessarily disabled warnings